decisions/wc_society_decisions.txt

```diff
@@ -917,4 +917,60 @@ society_decisions = {
 			# factor = 0
 		# }
 	# }
+	forge_hammer_of_the_twilight = {
+		is_high_prio = yes
+
+		ai_check_interval = 24
+
+		only_playable = yes
+		is_in_society = yes
+
+		potential = {
+			is_untouchable_trigger = no
+
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+
+			society_member_of = twilights_hammer_cult
+			society_rank == 4
+		}
+
+		allow = {
+			is_inaccessible_trigger = no
+
+			religion = old_gods_worship
+			true_religion = old_gods_worship
+
+			society_currency >= 5000
+			OR = {
+				AND = {
+					has_character_modifier = follower_of_nzoth
+					offmap_nzoth = { has_policy = old_god_working }
+				}
+				AND = {
+					has_character_modifier = follower_of_cthun
+					offmap_cthun = { has_policy = old_god_working }
+				}
+				AND = {
+					has_character_modifier = follower_of_yogg_saron
+					offmap_yoggsaron = { has_policy = old_god_working }
+				}
```

Comment:
You never will be able to get this hammer, yet, it show ups in decision menu, moreover, in high priority.
It's just player trolling.

Reply:
again, I didn't work on decisions yet.

Reply:
@Sejtalk please respond

Reply:
decision disabled